frappe/desk/form/load.py

```diff
@@ -226,7 +226,7 @@ def get_assignments(dt, dn):
 			filters={
 				'reference_type': dt,
 				'reference_name': dn,
-				'status': ('!=', 'Cancelled'),
+				'status': ('not in', ['Closed', 'Cancelled']),
```

Comment:
should be `('!=', 'Cancelled')` because... [read this](https://github.com/frappe/frappe/pull/8420#issue-317253845)